{
  "machine_id": "vm_001",
  "django_url": "http://localhost:8000",
  "status": "running",
  "mode": "TESTE",
  "connection_status": "disconnected",
  "inspection_config": {
    "tools": [
      {
        "ROI": {
          "h": 480,
          "w": 752,
          "x": 0,
          "y": 0
        },
        "id": 1,
        "method": "luminance",
        "name": "grayscale_filter",
        "normalize": false,
        "type": "grayscale"
      },
      {
        "ROI": {
          "h": 400,
          "w": 700,
          "x": 25,
          "y": 25
        },
        "area_max": 100000,
        "area_min": 0,
        "blob_count_test": true,
        "id": 2,
        "inspec_pass_fail": true,
        "name": "blob_1",
        "test_blob_count_max": 8,
        "test_blob_count_min": 8,
        "test_total_area_max": 600,
        "test_total_area_min": 50,
        "th_max": 255,
        "th_min": 60,
        "total_area_test": false,
        "type": "blob",
        "contour_chain": "TC89_KCOS",
        "approx_epsilon_ratio": 0.003,
        "polygon_max_points": 1000
      }
    ]
  },
  "source_config": {
    "type": "pasta",
    "camera_id": 0,
    "resolution": [
      752,
      480
    ],
    "fps": 30,
    "folder_path": "./testblob_images",
    "rtsp_url": ""
  },
  "trigger_config": {
    "type": "continuous",
    "interval_ms": 1000
  },
  "error_msg": "Erro ao inicializar source de imagem: Não foi possível abrir câmera 0",
  "last_saved": "2025-08-25T21:04:03.379746"
}